src/index.jsx

```diff
@@ -4,32 +4,33 @@ import 'regenerator-runtime/runtime';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import {
-  Route,
-  Switch,
-} from 'react-router-dom';
-import { Provider } from 'react-redux';
-import { ConnectedRouter } from 'react-router-redux';
-import { Helmet } from 'react-helmet';
-import { PrivateRoute } from '@edx/frontend-auth';
-import * as FullStory from '@fullstory/browser';
+  initialize, APP_INIT_ERROR, APP_READY, subscribe,
+} from '@edx/frontend-platform';
+import { ErrorPage } from '@edx/frontend-platform/react';
 
-import EnterpriseApp from './containers/EnterpriseApp';
-import Header from './containers/Header';
-import Footer from './containers/Footer';
-import EnterpriseIndexPage from './containers/EnterpriseIndexPage';
-import AdminRegisterPage from './containers/AdminRegisterPage';
-import UserActivationPage from './containers/UserActivationPage';
-import JwtAuthenticationRefresh from './components/JwtAuthenticationRefresh';
-import NotFoundPage from './components/NotFoundPage';
-import SupportPage from './components/SupportPage';
-import { withErrorBoundary } from './components/ErrorBoundary';
-import { ToastsProvider, Toasts } from './components/Toasts';
+import * as FullStory from '@fullstory/browser';
 
-import apiClient from './data/apiClient';
-import store from './data/store';
-import history from './data/history';
 import { configuration } from './config';
 
+import App from './components/App';
+
+subscribe(APP_READY, () => {
+  ReactDOM.render(<App />, document.getElementById('root'));
+});
+
+subscribe(APP_INIT_ERROR, (error) => {
+  ReactDOM.render(<ErrorPage message={error.message} />, document.getElementById('root'));
+});
+
+initialize({
+  messages: [],
+  // We don't require authenticated users so that we can perform our own auth redirect to a proxy login that depends on
+  // the route, rather than the LMS like frontend-platform does.
+  requireAuthenticatedUser: false,
+  hydrateAuthenticatedUser: true,
+});
+
+// eslint-disable-next-line import/first
 import './index.scss';
```

Comment:
Should this import be at the top of the file?

Reply:
I suspect it needs to be imported after the app is initialized? It's that way in learner portal, so I'm not gonna mess with it for now.

Reply:
Weird, sounds good to me haha.